### PR TITLE
Saleor 1612/add order confimation

### PR DESCRIPTION
**Pull request by mmarkusik**
I want to merge this change because it adds order confirm feature to order details. Also adds order confirmed to order history. 

[TICKET](https://app.clickup.com/t/2549495/SALEOR-1612)

**PR intended to be tested with API branch:** 

### Pull Request Checklist

1. [ ] All visible strings are translated with proper context.
1. [ ] All data-formatting is locale-aware (dates, numbers, and so on).
1. [ ] Translated strings are extracted.
1. [ ] Number of API calls is optimized.
1. [ ] The changes are tested.
1. [ ] Data-test are added for new elements.
1. [ ] Type definitions are up to date.
1. [ ] Changes are mentioned in the changelog.

### Test environment config

API_URI=https://master.staging.saleor.rocks/graphql/

### CODE REVIEW

**Comment by patrys** [comment]
Task linked: [SALEOR-1612 Add possibility to confirm an order through dashboard](https://app.clickup.com/t/2549495/SALEOR-1612)

**Review by orzechdev — APPROVED** [review]

File: src/orders/components/OrderDetailsPage/OrderDetailsPage.tsx [hunk 1]
```diff
@@ -136,6 +136,19 @@ const OrderDetailsPage: React.FC<OrderDetailsPageProps> = props => {
     privateMetadata: order?.privateMetadata.map(mapMetadataItemToInput)
   };
 
+  const getSaveLabel = () =>
+    order?.status === OrderStatus.UNCONFIRMED
+      ? intl.formatMessage({ defaultMessage: "confirm order" })
```
Comment (dominik-zeglen) [inline src/orders/components/OrderDetailsPage/OrderDetailsPage.tsx:141]: Missing description

File: src/orders/components/OrderDetailsPage/OrderDetailsPage.tsx [hunk 2]
```diff
@@ -136,6 +136,19 @@ const OrderDetailsPage: React.FC<OrderDetailsPageProps> = props => {
     privateMetadata: order?.privateMetadata.map(mapMetadataItemToInput)
   };
 
+  const getSaveLabel = () =>
```
Comment (dominik-zeglen) [inline src/orders/components/OrderDetailsPage/OrderDetailsPage.tsx:139]: I don't see the reason why this should be a function

File: src/orders/views/OrderDetails/index.tsx [hunk 3]
```diff
@@ -287,7 +307,11 @@ export const OrderDetails: React.FC<OrderDetailsProps> = ({ id, params }) => {
                           onInvoiceSend={id =>
                             openModal("invoice-send", { id })
                           }
-                          onSubmit={handleSubmit}
+                          onSubmit={
+                            order?.status === OrderStatus.UNCONFIRMED
+                              ? handleConfirmationSubmit
+                              : handleSubmit
+                          }
```
Comment (dominik-zeglen) [inline src/orders/views/OrderDetails/index.tsx:314]: Can we move this logic to submit handler and decide there whether order should be confirmed or not?

**Review by dominik-zeglen — CHANGES_REQUESTED** [review]

**Review by krzysztofwolski — APPROVED** [review]

**Pull request merged**